chapters/annotations.tex

```diff
@@ -1812,7 +1713,7 @@ \subsection{Version Date and Build Information}\label{version-date-and-build-inf
 \end{lstlisting}
 \end{example}
 
-The meaning of these annotation is:
+The meaning of these annotations is:
```

Comment:
Would it make more sense to write 'The meanings of these annotations are:'?

Reply:
I would think that the current way of writing would be the correct one if the different annotations have the following meaning when considered together; one meaning coming out of several parts.  Here, I agree with your intuition that this isn't just a single meaning.

Changed.  Resolving.